### ISSUE TITLE
Add ability to sort the x or y axis

### ISSUE THREAD

**Issue by cjhendrix** [OCHA-DAP/hdx-ckan#2646]
For data that is not a timeline, we may want to be able to sort (asc or desc) by values in the x or y axes.  For example, the 2nd mali chart would benefit from alphabetical sorting of the districts or numeric sorting of the values.  

![image](https://cloud.githubusercontent.com/assets/1654485/7115729/5cd16bda-e1eb-11e4-9e26-f3ed7677b6bc.png)

All graph styles except horizontal bar (which we have not enabled yet) will sort along X axis.

**Comment by danmihaila** [comment]
@cjhendrix I agree we might need this option. Lets think how to include a menu for all these filtering, etc.
